src/libs/wds-status.test.ts

```diff
@@ -118,7 +118,6 @@ describe('useWdsStatus', () => {
         wds: 'https://lz34dd00bf3fdaa72f755eeea8f928bab7cd135043043d59d5.servicebus.windows.net/wds-6601fdbb-4b53-41da-87b2-81385f4a760e-6601fdbb-4b53-41da-87b2-81385f4a760e/',
       },
       appName: 'wds-6601fdbb-4b53-41da-87b2-81385f4a760e',
-      // @ts-expect-error Leo app types do not include WDS
```

Comment:
Hmm... is this correct? You've added `WDS` to `AppToolLabel` type which should be sufficient to cause no errors here?

Reply:
I had to add this comment when I added this mock in an earlier PR. But this PR is removing the `ts-expect-error`.

Reply:
Ugh ok I need more coffee 🤦